app/components/SettingsCellComponent.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import {
+  View,
+  Text,
+  StyleSheet,
+  TouchableOpacity,
+  Dimensions,
+  Switch,
+} from 'react-native'
+import { colors } from '../config'
+
+const { settingsCellTextColor, settingsCellBorderLineColor } = colors
+const { width: screenWidth } = Dimensions.get('screen')
```

Comment:
We shouldn't need screen width either. I think we should be using `flex: 1` to make elements go full width/height (depending on flex direction)  then apply `margin` as needed.

Reply:
Done